OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -56,6 +56,8 @@ public class BuildingCategories
 			public readonly HashSet<string> Production = new HashSet<string>();
 			public readonly HashSet<string> NavalProduction = new HashSet<string>();
 			public readonly HashSet<string> Silo = new HashSet<string>();
+			public readonly HashSet<string> Radar = new HashSet<string>();
```

Comment:
Since these are only used for enemy building detection, IMO it'd be better if there would be a AdditionalBaseStructures or akin list instead of these two entries (The Temple of Nod is a fourth tier structure pretty much, as well as D2k).